### PR TITLE
Hide from committee

### PR DESCRIPTION
**Pull request by mixcas**
Hide actions buttons and status for non admin users (comittee). Also lock the subscription to only Paid applications.

### CODE REVIEW

File: app/imports/components/applications/applicationsLayout.jsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ export class ApplicationsFilters extends Component {
     let search = this.refs.search.value;
 
     // Search clears the status filter
-    if (search) {
+    if (search && isAdmin) {
```
Comment (varffs) [inline app/imports/components/applications/applicationsLayout.jsx]: shouldnt this be `props.isAdmin`?

File: app/imports/startup/server/publications.js [hunk 2]
```diff
@@ -142,6 +142,11 @@ Meteor.publishComposite('applications.index', function(posts, page, status, sort
   };
 
 
+  // Lock Committee members to Paid Applications
+  if (Roles.getUsersInRole('committee')) {
```
Comment (varffs) [inline app/imports/startup/server/publications.js]: this is the wrong condition and returns and array thus true. Should be `userIsInRole`

**Review by varffs — CHANGES_REQUESTED** [review]

File: app/imports/startup/server/publications.js (hunk 2 above)
Comment (mixcas) [reply app/imports/startup/server/publications.js]: oh true

File: app/imports/components/applications/applicationsLayout.jsx (hunk 1 above)
Comment (mixcas) [reply app/imports/components/applications/applicationsLayout.jsx]: true

**Comment by mixcas** [comment]
@varffs can u fix thos?

**Comment by varffs** [comment]
ok

**Review by varffs — APPROVED** [review]

**Comment by varffs** [comment]
@mixcas yours to close

**Pull request merged**